src/Symfony/Component/ExpressionLanguage/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.1.0
+-----
+
+ * Support lexing numbers with the `_` separator and decimals with no leading zero
```

Comment:
I suggest making that 2 changelog entries instead of using a `and`, for better visibility.